support-frontend/assets/components/orderSummary/contributionsOrderSummary.tsx

```diff
@@ -13,14 +13,18 @@ import {
 import { useState } from 'react';
 import type { CheckListData } from 'components/checkmarkList/checkmarkList';
 import { CheckmarkList } from 'components/checkmarkList/checkmarkList';
+import { CheckoutTopUpAmounts } from 'components/checkoutTopUpAmounts/checkoutTopUpAmounts';
+import { CheckoutTopUpAmountsContainer } from 'components/checkoutTopUpAmounts/checkoutTopUpAmountsContainer';
 import type { ContributionType } from 'helpers/contributions';
+import { simpleFormatAmount } from 'helpers/forms/checkouts';
+import type { Currency } from 'helpers/internationalisation/currency';
 
 const componentStyles = css`
 	${textSans.medium()}
 `;
 
-const summaryRow = css`
-	display: flex;
+const summaryRow = (withFlexWrap?: boolean) => css`
```

Comment:
```suggestion
const summaryRow = (withFlexWrap = false) => css`
```
Very nitpicky, but I think it's preferable to always have a known type and use a default parameter than to have to deal with the ambiguity of `T | undefined` if we don't absolutely have to.